package.json

```diff
@@ -40,6 +40,7 @@
     "crypto-js": "^4.2.0",
     "ffc-ahwr-event-publisher": "1.0.2",
     "ffc-messaging": "^2.9.1",
+    "form-data": "^4.0.1",
```

Comment:
Is this needed?

Reply:
Yeah its used in app/auth/auth-code-grant/redeem-authorization-code-for-access-token.js and app/auth/client-credential-grant/retrieve-apim-access-token.js .

Bizarrely its a dependency which we were using but not declaring in our package.json - we were using it secondhandedly from a dependency we werent using.